code/game/objects/items/fireaxe.dm

```diff
@@ -70,3 +70,18 @@
 
 /obj/item/fireaxe/boneaxe/update_icon_state()
 	icon_state = "bone_axe0"
+
+/*
+ * Metal Hydrogen Axe
+ */
+/obj/item/fireaxe/metalh2axe  // Blatant imitation of the fireaxe, but made out of metallic hydrogen
+	icon_state = "metalh2_axe0"
+	name = "metallic hydrogen axe"
+	desc = "A large, menacing axe made of an unkown substance that the most elder atmosians calls Metallic Hydrogen. Truly an otherwordly weapon."
+
+/obj/item/fireaxe/metalh2axe/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_unwielded=6, force_wielded=26, icon_wielded="metalh2_axe1")
```

Comment:
If this exists it should be a strict downgrade, fireaxe bones are hellish enough as it is.

Reply:
You think? I based the damage done on the difficulty of making enough metallic hydrogen to make one in a timely matter, if not more damage i think at least the same would be good

Reply:
I'm less worried about 1 guy getting it then I am about 1 guy making 10 for his friends. I suppose that's a pretty big feat in the first place, sorry if I'm over reacting

Reply:
well if you can make 10 of these it is a big feat, but i'll change the damage to be equal to the bone axe in another commit